src/mobility/components/OperatorBenefit.tsx

```diff
@@ -37,16 +45,11 @@ export const OperatorBenefit = ({
       <Section>
         <GenericSectionItem>
           <View style={style.benefitContainer}>
-            <View style={style.benefitImage}>
-              {isUserEligible && benefit.imageWhenActive && (
-                <SvgXml xml={benefit.imageWhenActive} />
-              )}
-              {!isUserEligible && benefit.imageWhenNotActive && (
-                <SvgXml xml={benefit.imageWhenNotActive} />
-              )}
-            </View>
+            <BenefitImage eligible={isUserEligible} formFactor={formFactor} />
             <View style={style.benefitContent}>
```

Comment:
The `marginStart` on `benefitContainer` will create an unwanted margin to the left of the text if `<BenefitImage>` is null.

Reply:
moved the margin to the image so if it's null there will be no margins.